airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -92,6 +97,25 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
               .withOperatorDbt(standardSyncOperation.getOperatorDbt());
 
           dbtTransformationActivity.run(jobRunConfig, destinationLauncherConfig, syncInput.getResourceRequirements(), operatorDbtInput);
+        } else if (standardSyncOperation.getOperatorType() == OperatorType.WEBHOOK) {
+          LOGGER.info("running webhook operation");
+          LOGGER.debug("webhook operation input: {}", standardSyncOperation);
+          boolean success = webhookOperationActivity
+              .invokeWebhook(new OperatorWebhookInput()
+                  .withExecutionUrl(standardSyncOperation.getOperatorWebhook().getExecutionUrl())
+                  .withExecutionBody(standardSyncOperation.getOperatorWebhook().getExecutionBody())
+                  .withWebhookConfigId(standardSyncOperation.getOperatorWebhook().getWebhookConfigId())
+                  .withWorkspaceWebhookConfigs(syncInput.getWebhookOperationConfigs()));
+          LOGGER.info("webhook {} completed {}", standardSyncOperation.getOperatorWebhook().getWebhookConfigId(),
+              success ? "successfully" : "unsuccessfully");
+          if (syncOutput.getWebhookOperationSummary() == null) {
+            syncOutput.withWebhookOperationSummary(new WebhookOperationSummary());
```

Comment:
does stubbing this in also make sure `getSuccesses` and `getFailures` never runs into a NPE?

Reply:
> does stubbing this in also make sure `getSuccesses` and `getFailures` never runs into a NPE?

Yep that's the idea here.

> is this actually the only place populating this summary?

It is the only place, but it's called in a loop. So the first iteration I expect `syncOutput.getWebhookOperationSummary()` to be null, and then the next time through it will be populated; so this seemed like a ~reasonable way to handle that.

Reply:
Apart from here, I don't see a place where WebhookOperationSummary is populated on SyncOutput, which means this is always stubbed.

I might be missing something though!

Reply:
Discussed offline, will plan to cleanup but leave this for now with a TODO and comment.